src/config.rs

```diff
@@ -70,6 +70,8 @@ pub struct GitDependency {
     pub version: String,
     pub git: String,
     pub rev: Option<String>,
+    pub tag: Option<String>,
+    pub branch: Option<String>,
```

Comment:
This would be better represented as an enum, seeing how it's only valid to have one of the three